packets/seedlings/src/_height.scss

```
@@ -78,10 +78,10 @@ Breakpoints:
     @media #{$breakpoint} {
       @include height($breakpoint-name);
       @include percent-height($breakpoint-name);
-      .h-auto {
+      .h-auto#{$breakpoint} {
```

I think this should be `$breakpoint-name`, right?

```suggestion
      .h-auto#{$breakpoint-name} {
```

---

🤦‍♂️ yes. fix pushed up, @ericcecchi